Dockerfiles/README.md

```diff
@@ -0,0 +1,50 @@
+# operator-bundle-validate-container
+CVP midstream operator bundle validate container
+
+# Usage
+
+## Building image on local machine 
+
+```
+git clone https://github.com/redhat-operator-ecosystem/operator-bundle-validate-container
```

Comment:
Should this just be `cd to Dockerfiles/` ?

Reply:
done